### DISCUSSION TITLE
Conda-forge CDN cloning offline?

### DISCUSSION THREAD

**Discussion by chrisburr** [conda/infra#6] [General]
A build of [`libudev`](https://anaconda.org/conda-forge/libudev) was uploaded to conda-forge over 100 minutes ago and it hasn't shown up on the CDN yet.

I've also checked the rss.xml but it appears to be corrupt and only contains the tail of the file.

Can someone take a look please?

Update: The build has appeared after about 2.5 hours, is the CDN just running more slowly these days? Though the XML is still invalid.

**Comment by jezdez** [comment]
Hey @chrisburr, yeah this isn't particularly fast, we need to look into what is causing those delays. I actually will convert this into a ticket to trigger the sync to a Jira ticket.

About the rss.xml, do you mean https://repo.anaconda.com/pkgs/rss.xml?

**Reply by isuruf** [reply]
conda-forge rss.xml is at https://conda-static.anaconda.org/conda-forge/rss.xml which is not a valid XML anymore.

**Reply by jezdez** [reply]
We should file a ticket for this I think.

**Comment by jakirkham** [comment]
We are seeing this issue again.

For example Python 3.10 builds of `python-javabridge` were uploaded ~2hrs ago, but conda search can't find it.

```
$ conda search "python-javabridge[channel=conda-forge]=4.0.3"
Loading channels: done
# Name                       Version           Build  Channel             
python-javabridge              4.0.3  py39h0d85af4_0  conda-forge
```

<img width="1792" alt="Screen Shot 2021-11-08 at 11 27 27 AM" src="https://user-images.githubusercontent.com/3019665/140805949-34d8389f-9903-4552-8bf7-33750f13fb5c.png">

**Reply by jakirkham** [reply]
Also seeing the same issue Isuru described above with ( https://conda-static.anaconda.org/conda-forge/rss.xml ) being invalid XML

<img width="1792" alt="Screen Shot 2021-11-08 at 11 33 48 AM" src="https://user-images.githubusercontent.com/3019665/140806160-7f57f4b3-a94d-4d14-a17b-372736e2da99.png">

**Reply by beckermr** [reply]
@jezdez @chenghlee for viz

**Reply by jakirkham** [reply]
Looks like this is working now

```
$ conda search "python-javabridge[channel=conda-forge]=4.0.3"
Loading channels: done
# Name                       Version           Build  Channel             
python-javabridge              4.0.3 py310h0d85af4_1  conda-forge         
python-javabridge              4.0.3  py39h0d85af4_0  conda-forge
```

**Reply by jezdez** [reply]
This was due to a CDN sync issue on Anaconda's side and was fixed in the meantime.
